src/backend/utils/storage.js

```diff
@@ -69,13 +75,23 @@ module.exports = {
    */
   removeFeed: async (id) => {
     const key = createFeedKey(id);
-    await redis
-      .multi()
-      .hdel(key, 'id', 'author', 'url', 'user', 'link', 'etag', 'lastModified')
-      .srem(feedsKey, id)
-      .exec();
+    // Checks which set the feed is currently in
+    const redisKey = (await redis.sismember(feedsKey, id)) ? feedsKey : flaggedFeedsKey;
+    try {
+      await redis
+        .multi()
+        .hdel(key, 'id', 'author', 'url', 'user', 'link', 'etag', 'lastModified')
+        .srem(redisKey, id)
+        .exec();
+    } catch (error) {
+      logger.error({ error }, `Error removing Feed ${id} from Redis`);
```

Comment:
Should we not throw too?  I'm not sure if eating a failure is a good idea.  Log it, yes, but throw too.

Reply:
Added a throw on line 89, let me know if that is what you had in mind